tee-worker/omni-executor/binance-api/src/lib.rs

```diff
@@ -194,12 +202,54 @@ impl BinanceApi for BinanceApiClient {
 	}
 }
 
+#[async_trait]
+impl BinancePaymasterApi for BinanceApiClient {
+	async fn get_symbol_price(&self, symbol: &str) -> Result<String, Error> {
+		use crate::spot_trading_api::types::SymbolPrice;
+
+		let endpoint = "/api/v3/ticker/price";
```

Comment:
I've just realized we already have methods for these endpoints in the `SpotTradingApi`